tests/fast/EncryptedBackupCorrectness.toml

```diff
@@ -1,5 +1,4 @@
 [configuration]
-allowDefaultTenant = true
```

Comment:
what failure do we have if we enable tenants in this test?

Reply:
There are no failures this option is just redundant, its true by default